test/navigation/FundRaisingNavigatorSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import base.SpecBase
+import controllers.routes
+import models._
+import models.operations.FundRaisingOptions
+import pages.IndexPage
+import pages.operationsAndFunds.FundRaisingPage
+
+class FundRaisingNavigatorSpec extends SpecBase {
+
+  val navigator = new FundRaisingNavigator()
+
+  "Navigator.nextPage(page, mode, userAnswers)" when {
+
+    "in Normal mode" when {
+
+      "from the FundRaisingPage page" must {
+
+        "go to the SessionExpiredController page when user answer is empty" in {
+          navigator.nextPage(FundRaisingPage, NormalMode, emptyUserAnswers) mustBe
+            routes.SessionExpiredController.onPageLoad()
+        }
+
+        "go to the Summary page when clicked continue button" in {
```

Comment:
the next page in the flow is the "Where does your charity operate?" page, it may make sense to have this in the TODO and test title rather than the summary.